extension/src/popups/MinervaMenu.html

```diff
@@ -14,74 +14,85 @@
     <script src="js/MinervaMenu.js"></script>
   </head>
   <body>
-    <div id="extensionBody">
-      <div class="collapse" id="minervaMenuCollapseCard">
-        <div class="card card-body">
-          <ul class="nav nav-pills justify-content-center">
-            <li class="nav-item">
-              <a class="nav-link active" id="you-tab" data-toggle="tab" href="#nav-you" role="tab" aria-controls="you"
-                aria-selected="true">You</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="ranking-tab" data-toggle="tab" href="#nav-ranking" role="tab" aria-controls="ranking"
-                aria-selected="false">Ranking</a>
-            </li>
-            <li class="nav-item">
-              <a class="nav-link" id="store-tab" data-toggle="tab" href="#nav-store" role="tab" aria-controls="store"
-                aria-selected="false">Store</a>
-            </li>
-          </ul>
-          <div class="tab-content" id="nav-tabContent">
-            <div class="tab-pane fade show active overflow-auto" id="nav-you" role="tabpanel" aria-labelledby="nav-you-tab">
-              <div class="card">
-                <div class="card-body">
-                  <h5 class="card-title" id="studentName-youPanel" ></h5>
-                  <p class="card-text">Total minutes watched: 69:42</p>
+    <div id="extensionBody" class="container">
+      <div class="row">
+        <div class="card-body" id="side-bar-owned-badges"></div>
+      </div>
+      <div class="row">
+        <div class="collapse" id="minervaMenuCollapseCard">
+          <div class="container">
+            <div class="card-body border-0">
+              <ul class="nav nav-pills justify-content-center">
+                <li class="nav-item">
+                  <a class="nav-link active" id="you-tab" data-toggle="tab" href="#nav-you" role="tab" aria-controls="you"
+                    aria-selected="true">You</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" id="ranking-tab" data-toggle="tab" href="#nav-ranking" role="tab" aria-controls="ranking"
+                    aria-selected="false">Ranking</a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" id="store-tab" data-toggle="tab" href="#nav-store" role="tab" aria-controls="store"
+                    aria-selected="false">Store</a>
+                </li>
+              </ul>
+              <div class="tab-content" id="nav-tabContent">
+                <div class="tab-pane fade show active overflow-auto" id="nav-you" role="tabpanel" aria-labelledby="nav-you-tab">
+                  <div class="card border-0">
+                    <h5 class="card-title" id="studentName-youPanel"></h5>
+                    <div class="container">
```

Comment:
container only need to exist once, at the outermost layer for the grid system to work